### PR TITLE
Ensure certificates are present for www and naked url

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
We had some reports of browsers intermittently failing to load the homepage when using the naked url `openfoodnetwork.org.uk`. At first it seemed to just be in Internet Explorer :nauseated_face: but it turned out to be intermittently affecting Firefox and Chrome as well. It turned out to be quite a critical issue, as multiple users were finding the site didn't load at all!

This seems to have resolved the problem on UK prod. It looks like France needs this as well, and probably others. We should double-check Katuma prod when provisioning this, as it has a setup with custom domains.

### CODE REVIEW

**Comment by Matt-Yorkley** [comment]
It's not looking good on UK Staging... needs further investigation.

**Comment by Matt-Yorkley** [comment]
OK, it took a while but I think I've finally got this fixed. Tested on UK staging and prod.

**Comment by Matt-Yorkley** [comment]
On Katuma it looks like you've added a manual nginx entry as a workaround for this:

katuma_nginx

This PR should remove the need for your workaround, and make it available to others.

**Comment by sauloperez** [comment]
I remember now. Done in https://github.com/coopdevs/katuma-provisioning/pull/14/files. I desperately want to :fire: this repo :sob:

**Comment by Matt-Yorkley** [comment]
I'm 95% sure this will be compatible with Katuma, but I think we can only test it by applying it to a live server...

**Comment by lin-d-hop** [comment]
Interestingly the API on UK Prod only works on the bare URL and not with the www. No idea why that might be but thought I'd flag. Any thoughts @matt-yorkley

**Comment by Matt-Yorkley** [comment]
> Interestingly the API on UK Prod only works on the bare URL and not with the www. No idea why that might be but thought I'd flag. Any thoughts @Matt-Yorkley

Maybe we provisioned again. This hasn't been merged yet.

**Comment by lin-d-hop** [comment]
@matt-yorkley Any testing tips here? Is this something that @rachel and I can test or does this need a sysadmin to reprovision?

File: inventory/host_vars/www.openfoodnetwork.org.uk/config.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ unicorn_timeout: 120
 
 swapfile_size: 1G
 
+certbot_domains:
+  - www.openfoodnetwork.org.uk
+  - openfoodnetwork.org.uk
+
```
Comment (enricostano) [inline inventory/host_vars/www.openfoodnetwork.org.uk/config.yml]: I don't understand the need of a second certificate. Wouldn't a redirect at nginx level do the job?

Comment (Matt-Yorkley) [reply inventory/host_vars/www.openfoodnetwork.org.uk/config.yml]: It adds both domains to a single certificate. It's more useful for subdomains.

**Comment by Matt-Yorkley** [comment]
@lin-d-hop it needs sysadmin testing.

@sauloperez shall we try this out on Katuma staging, and then production?

File: inventory/host_vars/www.openfoodnetwork.org.uk/config.yml (hunk 1 above)
Comment (enricostano) [reply inventory/host_vars/www.openfoodnetwork.org.uk/config.yml]: I don't like the `www` in the URL basically :joy: and I would not add a subdomain that would not be used to a certificate. But that's just me maybe. Don't bother, move on :muscle:

Comment (sauloperez) [reply inventory/host_vars/www.openfoodnetwork.org.uk/config.yml]: Mmm that's actually the standard way to deal with this common issue isn't it @enricostano? What do experts do?🤣

Comment (Matt-Yorkley) [reply inventory/host_vars/www.openfoodnetwork.org.uk/config.yml]: Confirmed Katuma had the same issue as UK:

FAIL

and the issue is now fixed. :tada:

**Pull request merged**